control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1286,6 +1289,16 @@ func (r *reconciler) reconcileOLM(ctx context.Context, hcp *hyperv1.HostedContro
 
 	p := olm.NewOperatorLifecycleManagerParams(hcp)
 
+	// Check if the defaultSources are disabled
+	operatorHub := &configv1.OperatorHub{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: OperatorHubObjectValue,
```

Comment:
I find it confusing to name a variable that contains the name `OperatorHubObjectValue`, I think we can just inline the name here